src/schema/rules/checks/electrodes.yaml

```diff
@@ -0,0 +1,15 @@
+---
+ElectrodeSpecificity:
+  issue:
+    code: EXCESSIVE_ELECTRODE_SPECIFICITY
+    message: |
+      Run or acquisition entities detected in electrodes.tsv.
+      Electrode definitions should generally not vary within a session.
+      Consider creating a new session each time electrodes are reconfigured.
+    level: warning
+  selectors:
+    - suffix == 'electrodes'
+    - extension == '.tsv'
+  checks:
+    - '!("run" in entities)'
+    - '!("acquisition" in entities)'
```

Comment:
I would take the "acq" entity out of this check, as "pre" and "post" measurements (using `acq-pre` and `acq-post`) for example would be an understandable use-case.

Reply:
This is a warning, not an error, so people are free to decide that their reasons for violating the recommendation are good.

I'm okay removing `acq` if you think it will generally be a legitimate use.

FWIW, here are the uses in OpenNeuro (removing redundant examples):

```
./ds003688/sub-30/ses-iemu/ieeg/sub-30_ses-iemu_acq-HDgrid_electrodes.tsv
./ds003688/sub-30/ses-iemu/ieeg/sub-30_ses-iemu_acq-clinical_electrodes.tsv
./ds004100/sub-HUP060/ses-presurgery/ieeg/sub-HUP060_ses-presurgery_acq-seeg_space-fsaverage_electrodes.tsv
./ds004100/sub-HUP064/ses-presurgery/ieeg/sub-HUP064_ses-presurgery_acq-ecog_space-fsaverage_electrodes.tsv
./ds004194/sub-p02/ses-umcuiemu01/ieeg/sub-p02_ses-umcuiemu01_acq-clinical_electrodes.tsv
./ds004194/sub-p03/ses-nyuecog01/ieeg/sub-p03_ses-nyuecog01_acq-clinical_electrodes.tsv
```

Reply:
These two are electrodes of ECoG and SEEG, apparently recorded together, but stored in two separate files. That is not what I would have expected, they should be in the same tsv file and be documented with a single coordsys.json.  

```
./ds004100/sub-HUP060/ses-presurgery/ieeg/sub-HUP060_ses-presurgery_acq-seeg_space-fsaverage_electrodes.tsv
./ds004100/sub-HUP064/ses-presurgery/ieeg/sub-HUP064_ses-presurgery_acq-ecog_space-fsaverage_electrodes.tsv
```

Reply:
Those are two different subjects. There aren't different acquisitions within a subject in that dataset.

Reply:
oh sorry, my bad!

Reply:
@sappelhoff Just checking in about whether you still think the `acq` check should be removed.

Reply:
I agree with your reasoning in https://github.com/bids-standard/bids-specification/pull/1722#discussion_r1520070303 and I am fine to keep the acq check. Thanks